pallets/committee/src/lib.rs

```diff
@@ -357,11 +357,7 @@ impl<T: Trait<I>, I: Instance> GroupTrait<T::Moment> for Module<T, I> {
 }
 
 impl<T: Trait<I>, I: Instance> ChangeMembers<IdentityId> for Module<T, I> {
-    fn change_members_sorted(
-        _incoming: &[IdentityId],
-        outgoing: &[IdentityId],
-        new: &[IdentityId],
-    ) {
+    fn change_members_sorted(incoming: &[IdentityId], outgoing: &[IdentityId], new: &[IdentityId]) {
```

Comment:
Maybe add a dev note for future that this function specifically uses `SystematicIssuers::GovernanceCommittee` as cdd issuer so if multiple instances of this were used of this module then a different implementation of these functions should be used (like we do in cdd providers).

Reply:
Comment added. I've also renamed `SystematicIssuers::GovernanceCommittee` to `SystematicIssuers::Committee`.